x11/tk/Portfile

```diff
@@ -19,11 +19,13 @@ platforms           darwin
 
 master_sites        sourceforge:project/tcl/Tcl/${version}
 dist_subdir         tcltk
-distname            ${name}${version}-src
+# Errata release 8.6.11.1
+#distname            ${name}${version}-src
+distname            ${name}${version}.1-src
```

Comment:
Is there a reason not to make the actual port version 8.6.11.1?

Reply:
I don't think there is a reason to use 8.6.11.1 as the tk port version. The corrected Tk distfile is named 8.6.11.1 merely to distinguish from the original Tk 8.6.11 distfile; upstream provided it as an alternative to them correcting the 8.6.11 distfile without renaming it, or them only offering a patch for package managers to apply to the original distfile. The version reported by Tk is still 8.6.11.

`port livecheck tk` detects 8.6.11 as latest version.

Reply:
OK